### PR TITLE
chore: configure tailwind to allow nested css in eds

### PR DESCRIPTION
**Pull request by schakraborty22**
### Summary:
This PR is to configure the tailwind to allow nested css in eds ,to handle the bunch of warnings in the dev console in EDS storybook complaining about tailwind not being configured to handle nested CSS.

### CODE REVIEW

**Comment by dierat** [comment]
Test plan is:
- run storybook
- open the browser dev console
- verify there are no warnings about tailwind not being configured to allow CSS nesting

File: postcss.config.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ module.exports = {
     'postcss-import': {
       addModulesDirectories: ['node_modules'],
     },
+    'tailwindcss/nesting': {},
     'postcss-mixins': {},
     'postcss-simple-vars': { variables: breakpoints },
     // TODO: replace with tailwindcss/nesting (https://tailwindcss.com/docs/using-with-preprocessors#nesting)
```
Comment (jinlee93) [inline postcss.config.js:11]: You can remove this todo comment now

[the remark this answers is not in the document]
Comment (dierat) [reply postcss.config.js:11]: I can't leave a comment there but can you check if removing line 12 brings the console warnings back? I think we can remove that line.

[the remark this answers is not in the document]
Comment (dierat) [reply postcss.config.js:11]: Oops, jinx @jinlee93

**Comment by jinlee93** [comment]
Is there value in an alpha publish and testing if styles are working as part of the package since this change would also enable TW outside of Storybook?

**Review by dierat — APPROVED** [review]

**Comment by dierat** [comment]
> Is there value in an alpha publish and testing if styles are working as part of the package since this change would also enable TW outside of Storybook?

I can do that.

**Pull request merged**